pkg/oam/util/helper.go

```diff
@@ -72,6 +73,12 @@ const (
 	errFmtInvalidRevisionType    = "invalid type of revision %s, type should not be %v"
 )
 
+type namespaceContextKey int
+
+const (
+	appDefinitionNamespace namespaceContextKey = iota
```

Comment:
```suggestion
	AppDefinitionNamespace namespaceContextKey = iota
```

maybe we need it to be a public var？

Reply:
Yes, this var maybe used outside the package in the future.